server/base/src/main/java/org/apache/accumulo/server/master/recovery/RecoveryPath.java

```diff
@@ -41,15 +47,14 @@ public static Path getRecoveryPath(Path walPath) {
 
       // drop wal
       walPath = walPath.getParent();
+      walPath = new Path(walPath, FileType.RECOVERY.getDirectory() + '-' + context.getInstanceID());
```

Comment:
Perhaps I am missing something here, but isn't context.getInstanceID() the accumulo instance id?  How does this make the directory any more unique?

Reply:
You're not missing anything. You're correct. This does not make it unique per tserver process.